### PR TITLE
GameDB: Add fixes for 'Shadow Man - 2econd Coming' and 'Ghosthunter' games.

### PR DESCRIPTION
**Pull request by Mrlinkwii**
### Description of Changes

adds 'EETimingHack' to 'Shadow Man - 2econd Coming'
### Rationale behind Changes

fixes issues 
### Suggested Testing Steps

look at CI test said game

### CODE REVIEW

**Comment by gilderoylockhart01** [comment]
Does the dma busy gamefix also work?

**Review by F0bes — APPROVED** [review]

**Comment by F0bes** [comment]
> Does the dma busy gamefix also work?

No, the cutscene issue persists when `EE timing hack` off and `Handle DMAC writes when it is Busy` on.

**Comment by LoStraniero91** [comment]
So far, so good. The PR fixes all the issues I've reported.

**Comment by F0bes** [comment]
Would you be able to change the commit & pr name to reflect the changes for the Ghosthunter games ?

`GameDB: Add fixes for 'Shadow Man - 2econd Coming' and 'Ghosthunter' games.` Would be suitable.

**Comment by Mrlinkwii** [comment]
> Would you be able to change the commit & pr name to reflect the changes for the Ghosthunter games ?
> 
> `GameDB: Add fixes for 'Shadow Man - 2econd Coming' and 'Ghosthunter' games.` Would be suitable.

done.

File: bin/GameIndex.yaml [hunk 1]
```diff
@@ -8105,6 +8114,8 @@ SLES-50446:
   name: "Shadow Man - 2econd Coming"
   region: "PAL-M4"
   compat: 5
+  gameFixes:
+    - EETimingHack # Fixes cutscenes freezes.
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:8118]: cutscene freezes

File: bin/GameIndex.yaml [hunk 2]
```diff
@@ -8364,6 +8375,8 @@ SLES-50608:
   name: "Shadow Man - 2econd Coming"
   region: "PAL-G"
   compat: 5
+  gameFixes:
+    - EETimingHack # Fixes cutscenes freezes.
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:8379]: same as above

File: bin/GameIndex.yaml [hunk 3]
```diff
@@ -1408,6 +1408,11 @@ SCED-51700:
   region: "PAL-M5"
   gameFixes:
     - VUKickstartHack # Fixes Character SPS.
+SCED-51922:
+  name: "Ghosthunter [Demo]"
+  region: "PAL-G"
+  clampModes:
+    vuClampMode: 3 # Fixes SPS
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:1415]: period

File: bin/GameIndex.yaml [hunk 4]
```diff
@@ -2378,6 +2383,8 @@ SCES-51463:
   name: "Ghosthunter"
   region: "PAL-M6"
   compat: 5
+  clampModes:
+    vuClampMode: 3 # Fixes SPS
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:2387]: period

File: bin/GameIndex.yaml [hunk 5]
```diff
@@ -2566,6 +2573,8 @@ SCES-52154:
 SCES-52156:
   name: "Ghosthunter - Hunter on the spectres"
   region: "PAL-M3"
+  clampModes:
+    vuClampMode: 3 # Fixes SPS
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:2577]: period

File: bin/GameIndex.yaml [hunk 6]
```diff
@@ -9367,6 +9380,8 @@ SLES-51113:
   name: "Zone of the Enders: The 2nd Runner Special Edition"
   region: "PAL-M5"
   compat: 5
+  gameFixes:
+    - DMABusyHack # Fixes the corrupted fade effect in cutscenes
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:9384]: period

File: bin/GameIndex.yaml [hunk 7]
```diff
@@ -34301,6 +34322,8 @@ SLUS-20413:
   name: "Shadow Man - 2econd Coming"
   region: "NTSC-U"
   compat: 5
+  gameFixes:
+    - EETimingHack # Fixes cutscenes freezes.
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:34326]: cutscene freezes, again

**Pull request merged**